README.md

```diff
@@ -157,10 +167,12 @@ ninja
 ls images
 # capdl-loader-image-arm-qemu-arm-virt
 ```
+
 #### Running seL4 qemu-arm-virt image
+
 ```sh
-./simulate 
-# ./simulate: qemu-system-aarch64 -machine virt,virtualization=on,highmem=off,secure=off -cpu cortex-a53 -nographic  -m size=2048  -kernel images/capdl-loader- image-arm-qemu-arm-virt 
+./simulate
+# ./simulate: qemu-system-aarch64 -machine virt,virtualization=on,highmem=off,secure=off -cpu cortex-a53 -nographic  -m size=2048  -kernel images/capdl-loader- image-arm-qemu-arm-virt
```

Comment:
```suggestion
# ./simulate: qemu-system-aarch64 -machine virt,virtualization=on,highmem=off,secure=off -cpu cortex-a53 -nographic  -m size=2048  -kernel images/capdl-loader-image-arm-qemu-arm-virt
```
Isn't there a space too much in the image name?

Reply:
yup, thanks for spotting that!